unittest/CMakeLists.txt

```diff
@@ -0,0 +1,40 @@
+
+include(ExternalProject)
+ExternalProject_Add(gtest
+                    GIT_REPOSITORY  https://github.com/google/googletest.git
+                    GIT_TAG         release-1.10.0
+                    SOURCE_DIR      "${CMAKE_BINARY_DIR}/gtest-src"
+                    BINARY_DIR      "${CMAKE_BINARY_DIR}/gtest-build"
+                    CMAKE_ARGS      ${CMAKE_REQUEST_PIC} ${CMAKE_EXTRA_GTEST_OPTS}
+                                    -DCMAKE_C_COMPILER=${CMAKE_C_COMPILER}
+                                    -DCMAKE_CXX_COMPILER=${CMAKE_CXX_COMPILER}
+                                    -DCMAKE_INSTALL_PREFIX=<INSTALL_DIR>
+                                    -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}
+                                    -DCMAKE_MAKE_PROGRAM=${CMAKE_MAKE_PROGRAM}
+                                    -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TOOLCHAIN_FILE}
+                    BUILD_COMMAND   ${CMAKE_COMMAND} --build .
+                    BUILD_BYPRODUCTS <BINARY_DIR>/lib/libgtest.a
+                    INSTALL_COMMAND ""
+                    TEST_COMMAND    "")
+
+ExternalProject_Get_Property(gtest SOURCE_DIR)
+set(GTEST_INCLUDE_DIR ${SOURCE_DIR}/include)
+ExternalProject_Get_Property(gtest BINARY_DIR)
+set(GTEST_LIBRARY_PATH ${BINARY_DIR}/lib/${CMAKE_FIND_LIBRARY_PREFIXES}gtest.a)
```

Comment:
You use `CMAKE_FIND_LIBRARY_PREFIXES` here, but not in `BUILD_BYPRODUCTS`? Also we don't use this anywhere else in lammps. Plus the name suggest that could be a list.

Reply:
done. thanks